### DISCUSSION TITLE
Import messes up Description on the 7th of very month?

### DISCUSSION THREAD

**Discussion by AngeloHulshout** [firefly-iii/firefly-iii#9547] [General]
Today I used Firefly for the first time, and imported my transactions for the this year. What I find is that it works fine, except that for the 7th of every month the description field is empty and the content is put in notes instead. I've been in software development for 30 years now, but this I cannot explain.... who can?

UPDATE: While correcting I noticed that all these transcations have their date wrong as well - they are transactions for July, the 7th month...

**Comment by AngeloHulshout** [comment]
chucks. Found it. I ran a test with only the month of July using a wrong data format, and apparently did not delete all entries before imporating the full year.

**Comment by JC5** [comment]
Nice, glad to hear you found it.
